kitsune/wiki/tests/test_templates.py

```diff
@@ -1620,6 +1620,30 @@ def test_self_approve_with_revision_contributors(self):
         assert rev1.creator.username in label
         assert u.username not in label
 
+    def test_review_passed_revision(self):
+        """Verify that its not possible to review a revision older than the current revision"""
+        r1 = revision(is_approved=False, save=True)
+        r2 = revision(document=r1.document, is_approved=True)
+        r2.save()
+        u = user(save=True)
+        add_permission(u, Revision, 'review_revision')
+        self.client.login(username=u.username, password='testpass')
+
+        response = get(self.client, 'wiki.review_revision',
+                       args=[r1.document.slug, r1.id])
+        eq_(200, response.status_code)
+        # While there is no unapproved revision after the current revision.
+        doc = pq(response.content)
+        doc_content = doc('.grid_9 #review-revision').text()
+        message1 = "A newer revision has already been reviewed."
+        message2 = "But there is another latest revision which is waiting for review."
+        assert message1 in doc_content
+        assert message2 not in doc_content
+        # While there is Unapproved revision after the Current Revision
+        r3 = revision(document=r1.document, is_approved=False)
+        r3.save
+        assert message1 and message2 in doc_content
```

Comment:
This is not a valid way to test if both messages are in doc_content. This tests that `message1` is truthy, and that `message2` is in doc_content. You can see this behavior with the expressions `4 and 1 in [0, 1, 2]` or `0 and 1 in [0, 1, 2]`.

The correct thing to do would be

```python
assert message1 in doc_content
assert message2 in doc_content
```